### ISSUE TITLE
ValueError: The number of weights does not match the population

### ISSUE THREAD

**Issue by JAYCHOU2020** [MCG-NJU/MixFormer#40]
How should this problem be solved, the environment configuration is the same as yours

**Comment by yutaocui** [comment]
Can you provide the detailed error information?

**Comment by wsasdsda** [comment]
Have you solved this problem?I meet the same question now!

**Comment by JAYCHOU2020** [comment]
> 

![image](https://user-images.githubusercontent.com/62323435/201295029-a216ca13-29a2-4aa7-8ef4-8507ff31e742.png)

**Comment by JAYCHOU2020** [comment]
> Can you provide the detailed error information?

i still cant fix it

**Comment by JAYCHOU2020** [comment]
> Can you provide the detailed error information?

![image](https://user-images.githubusercontent.com/62323435/201295262-2357dfba-a52b-47dc-b2ed-6f29eb027137.png)

**Comment by JAYCHOU2020** [comment]
> Can you provide the detailed error information?
![image](https://user-images.githubusercontent.com/62323435/201302702-cf7ba8a6-7c63-45c4-9f4c-966dd157ee99.png)

**Comment by JAYCHOU2020** [comment]
> Have you solved this problem?I meet the same question now!

no, have you fix it?

**Comment by laisimiao** [comment]
@JAYCHOU2020 
It means your number of datasets and your number of probabilities of your datasets are not same.

**Comment by JAYCHOU2020** [comment]
> @JAYCHOU2020 It means your number of datasets and your number of probabilities of your datasets are not same.

yes I'm only training one class "dog" of the lasot dataset, so where can I change the number?

**Comment by laisimiao** [comment]
See yaml file you used under experiment folder
![image](https://user-images.githubusercontent.com/37405764/201305134-d5e26cbd-d115-491e-87d0-a4b5e42e9971.png)

**Comment by JAYCHOU2020** [comment]
> yes I'm only training one class "dog" of the lasot dataset, so where can I change the nu

> See yaml file you used under experiment folder ![image](https://user-images.githubusercontent.com/37405764/201305134-d5e26cbd-d115-491e-87d0-a4b5e42e9971.png)


it works!thanks

**Comment by wsasdsda** [comment]
> > yes I'm only training one class "dog" of the lasot dataset, so where can I change the nu
> 
> > See yaml file you used under experiment folder ![image](https://user-images.githubusercontent.com/37405764/201305134-d5e26cbd-d115-491e-87d0-a4b5e42e9971.png)
> 
> it works!thanks

我在训练的时候一直卡在这个位置，也不报错，请问是什么原因啊？
![image](https://user-images.githubusercontent.com/109134157/201513131-8e6c3dd5-21d5-42c2-bc69-09eb0074b967.png)

**Comment by wsasdsda** [comment]
> See yaml file you used under experiment folder ![image](https://user-images.githubusercontent.com/37405764/201305134-d5e26cbd-d115-491e-87d0-a4b5e42e9971.png)

我在训练的时候一直卡在这个位置，也不报错，请问是什么原因啊？
![image](https://user-images.githubusercontent.com/109134157/201513252-e2374007-8a81-483a-8d1f-9df221dea696.png)
